### DISCUSSION TITLE
Welcome to gt4sd-core Discussions!

### DISCUSSION THREAD

**Discussion by drugilsberg** [GT4SD/gt4sd-core#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kwehden** [comment]
Excited to be here! Looking forward to sprucing up the place!

**Comment by palazzef** [comment]
Hi gt4sd folks, my name is Ferruccio (Ferri for everyone), and I am a compchem. I started to play with the gt4sd library and I found it super nice...so I have tons of questions for you :)

**Comment by runclezhan** [comment]
Does the gt4sd support Windows？

**Reply by drugilsberg** [reply]
We do not support Windows, but you can still use it on Windows via WSL: https://learn.microsoft.com/en-us/windows/wsl/install
